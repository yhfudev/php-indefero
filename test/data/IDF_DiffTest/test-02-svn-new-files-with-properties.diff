Index: LinuxBIOSv1/src/include/cpu/i786/cpufixup.h
===================================================================
--- LinuxBIOSv1/src/include/cpu/i786/cpufixup.h	(Revision 0)
+++ LinuxBIOSv1/src/include/cpu/i786/cpufixup.h	(Revision 665)
@@ -0,0 +1,11 @@
+#ifndef CPU_I786_CPUFIXUP_H
+#define CPU_I786_CPUFIXUP_H
+
+
+void i786_cpufixup(struct mem_range *mem);
+
+#define L3_CACHE_DISABLE 0x40
+
+#endif /* CPU_I786_CPUFIXUP_H */
+
+

Eigenschafts�nderungen: LinuxBIOSv1\src\include\cpu\i786\cpufixup.h
___________________________________________________________________
Hinzugef�gt: svn:keywords
   + Author Date Id Revision
Hinzugef�gt: svn:eol-style
   + native

