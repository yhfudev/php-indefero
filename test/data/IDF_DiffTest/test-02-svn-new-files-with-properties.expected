<?php return array (
  'LinuxBIOSv1/src/include/cpu/i786/cpufixup.h' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '',
          1 => '1',
          2 => '#ifndef CPU_I786_CPUFIXUP_H
',
        ),
        1 => 
        array (
          0 => '',
          1 => 2,
          2 => '#define CPU_I786_CPUFIXUP_H
',
        ),
        2 => 
        array (
          0 => '',
          1 => 3,
          2 => '
',
        ),
        3 => 
        array (
          0 => '',
          1 => 4,
          2 => '
',
        ),
        4 => 
        array (
          0 => '',
          1 => 5,
          2 => 'void i786_cpufixup(struct mem_range *mem);
',
        ),
        5 => 
        array (
          0 => '',
          1 => 6,
          2 => '
',
        ),
        6 => 
        array (
          0 => '',
          1 => 7,
          2 => '#define L3_CACHE_DISABLE 0x40
',
        ),
        7 => 
        array (
          0 => '',
          1 => 8,
          2 => '
',
        ),
        8 => 
        array (
          0 => '',
          1 => 9,
          2 => '#endif /* CPU_I786_CPUFIXUP_H */
',
        ),
        9 => 
        array (
          0 => '',
          1 => 10,
          2 => '
',
        ),
        10 => 
        array (
          0 => '',
          1 => 11,
          2 => '
',
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '0',
          1 => '0',
        ),
        1 => 
        array (
          0 => '1',
          1 => '11',
        ),
      ),
    ),
  ),
);
